### DISCUSSION TITLE
Adding products to meal plan

### DISCUSSION THREAD

**Discussion by Bovive** [firstof9/lovelace-grocy-meal-plan-card#27] [General]
If I add a product to a meal plan in Grocy the card will not display in Home Assistant. Recipes work just fine. If I remove the product from the meal plan the card shows back up again. I was curious if this is a problem anyone else has. Looking at the data from sensor.grocy_meal_plan the data for that product appears as:

```
state_class: measurement
meals: 
- id: 90
  day: '2023-01-05'
  recipe_id: null
  recipe_servings: 1
  note: null
  recipe: null
  section_id: 3
  section:
    id: 3
    name: Dinner
    sort_number: 3
    row_created_timestamp: '2022-11-29T18:19:54'
  type: product
  product_id: 46
  picture_url: null
```

It looks like it is not pulling in a name or much information on the product. Is this perhaps more of a problem with the integration? Thanks!

**Comment by firstof9** [comment]
If that's the data the sensor produces, that's going to be an issue with the integration, not the card.

**Reply by Bovive** [reply]
Thanks, I thought that might be the case. Is there any way to get the card to ignore the type "product" instead of disappearing?

**Reply by firstof9** [reply]
Since the integration is outputting `null` as the value of things, no.

**Reply by Bovive** [reply]
Thanks very much. I created an issue for the integration to see if products can be added. https://github.com/custom-components/grocy/issues/258
